tasks/rh-sign-cosign-image/tests/test-rh-sign-cosign-image-multiple-components.yaml

```diff
@@ -0,0 +1,122 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-rh-sign-cosign-image-multiple-components
+spec:
+  description: Test signing a single image with cosign 
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+              # empty mock calls files
```

Comment:
What's this comment? A leftover from something that was removed?

Reply:
fixed